### PR TITLE
Add scale property and allow to directly use composable without placing in hook

### PR DESCRIPTION
**Pull request by linspw**
Hello, the idea of ​​this PR is to add the possibility of scaling when calculating the size of the container

And together I added a watcher for the element and properties, to allow them to be reactive

The PR does not add breaking changes, all previous methods continue to work

### CODE REVIEW

File: src/components/resizable.tsx [hunk 1]
```diff
@@ -17,13 +17,9 @@ export const Resizable = defineComponent({
...
-    let init = false
-    onMounted(() => {
-      if (!wrapperRef.value || init)
-        return
-      useResizable(wrapperRef.value, props.config)
-      init = true
-    })
-    return () => h(props.as, { class: 'relative', ref: wrapperRef }, slots.default?.())
+
+    const result = useResizable(wrapperRef, toRef(props, 'config'))
```
Comment (linspw) [inline src/components/resizable.tsx:21]: More simple way to use in component

File: src/composables/resizable.ts [hunk 2]
```diff
@@ -9,20 +10,36 @@ function shouldRenderBorder(config: ResizableConfig['border']) {
   return (typeof config === 'object' && config.render) || config
 }
 
-export function useResizable(el: ResizableEl, resizableConfig: ResizableConfig) {
-  const config = parseConfig(resizableConfig)
+export function useResizable(el: MaybeElementRef<ResizableEl | null>, resizableConfig?: MaybeRefOrGetter<ResizableConfig>) {
+  const config = computed(() => {
+    return parseConfig(toValue(resizableConfig))
+  })
```
Comment (linspw) [inline src/composables/resizable.ts:16]: reactive calculated config

File: src/composables/resizable.ts [hunk 3]
```diff
@@ -9,20 +10,36 @@ function shouldRenderBorder(config: ResizableConfig['border']) {
...
+    handlePointerMove: () => {
+      if (!toValue(el))
+        return
+
+      size.value = updateSize({
+        el: toValue(el)!,
+        deltaPosition: toValue(payload!.deltaPosition),
+        type: toValue(payload!.moveType)!,
+        config: config.value,
+      })
```
Comment (linspw) [inline src/composables/resizable.ts:30]: returning the size, to be used in some case

**Comment by alexzhang1030** [comment]
Thank you very much!

**Pull request merged**